docs/root/modules/entra/schema.md

```diff
@@ -73,7 +73,7 @@ Representation of an Entra [User](https://learn.microsoft.com/en-us/graph/api/us
 - Entra users can have a manager
 
     ```cypher
-    (:EntraUser)-[:MANAGES]->(:EntraUser)
```

Comment:
This never existed before and probably was AI slop

Searching for "MANAGES" gets me: 
<img width="1303" height="1119" alt="Image" src="https://github.com/user-attachments/assets/74f432d5-e830-4243-860a-24b10c68d6af" />